### DISCUSSION TITLE
Multilingual translation using deepl

### DISCUSSION THREAD

**Discussion by twinko** [maheini/FreeStyle-Libre-3-patch#11] [Ideas]
Hey,

 i can translate the whole readme with deepl in a ton of languages. At least for the main languages deepl is so good, its hard to spot the difference to a human translation (it way better than google translater). Not sure about the japanese and chinese quality. Let me know if i should start the translation.

I would add a sentence in the beginning, that says something like:
This text is auto translated. You should read the English or German version for the latest updates.

I only would update the languages if major changes happen. I could translate to:
French
Italien
Russian
spanish
chinese
japanese
(feel free to ad an available language from deepl, if i forgat an important one)

At least in my xdrip related tutorials, spanish and russian translation are used a lot.

Regards

**Comment by maheini** [comment]
Thank you so much for your offer @twinko .

It's totally up to you, but I would highly appreciate it 😄. I can see you already know which languages would be needed. Surprisingly, turkish is very well translated within xdrip and androidaps - I guess it would be needed language as well.

If you know how to use Git/Github, just pull requests. If you have any questions - feel free to ask at any time.

Please note: I will add a new readme (with disclaimer) and full instructions within 2-3 days. I would recommend to wait until then.

**Reply by twinko** [reply]
awesome, ill add turkish to the language list,
And yes ill wait. If you think of it, feel free to leave me a comment here whenever you updated the disclaimer.

**Reply by twinko** [reply]
btw i definitly need help with the flags, not sure where do u get them :D

**Reply by maheini** [reply]
Ok, the repo is up to date now - feel free 😄.

Regarding the language flags:
1. Search for the appropriate flag in 4:3 format and download this image. You don't need to resize the image.
2. Upload this image to Github user content. You can do it by just drag&drop it to the readme (in edit mode). After, you will get the link for this image.
3. Now you can just paste the link within this boilerplate code:

```
<a href="LINK_TO_THE_README_WITH_LANGUAGE_CODE.de-de.md"><img  alt="LANGUAGE_CODE_LIKE_DE" src="PASTE_THE_LINK_FOR_YOUR_COUNTRY_FLAG_HERE" width="18px" /> CHANGE_THE_VISIBLE_TEXT_TO_LIKE_README_IN_ENGLISH</a>
```
